test/test_parser.cpp

```diff
@@ -5005,7 +5010,7 @@ TEST(Parser_Dictionary)
     verify_query(test_context, origin, "link.dict.Value > 50", 3);
     verify_query(test_context, origin, "links.dict['Value'] > 50", 5);
     verify_query(test_context, origin, "links.dict > 50", 6);
-    verify_query(test_context, origin, "links.dict['Value'] == NULL", 10);
+    verify_query(test_context, origin, "links.dict['Value'] == NULL", 1);
```

Comment:
If I'm reading this correctly, the meaning of the query `dict['Value'] == NULL` changes from "match if the dictionary doesn't have the key called 'Value'" to "match if the dictionary has the key called 'Value' and it's associated value is NULL". 

While I can see an argument for both sides, this is technically a breaking change. With these changes, is there any way to query for the existence of a key in a dictionary without specifying the value?

Reply:
Yes, I think the behavior was incorrect before. I can can create am issue and treat this as a fix.

Reply:
Great!